dashboard/src/components/api/che-workspace.factory.js

```diff
@@ -332,7 +332,17 @@ export class CheWorkspace {
   }
 
   getAllProjects() {
-    let projects = this.lodash.pluck(this.workspaces, 'config.projects');
+    let projects = this.lodash.map(this.workspaces, (workspace) => {
```

Comment:
are we forced to update 'model' representation with custom attributes ?
it's not possible to use workspace.id or workspace.config.name by the consumer ?